src/components/Panel/ChainIcon.tsx

```diff
@@ -0,0 +1,68 @@
+import { supportedChainsById } from "@/constants";
```

Comment:
I grabbed this from the voter dapp to use as a placeholder. We will fill this out with everything in a future pr